server/index.test.js

```diff
@@ -0,0 +1,11 @@
+var request = require('request');
+import React from 'react';
+import App from './client/src/components/App.jsx';
+import Server from './index.js';
```

Comment:
you can drop the `.jsx` and `.js` from the `import` statements bc your `webpack.config.js` automatically resolves `.js` and `.jsx` extensions